recipes/manorm/meta.yaml

```diff
@@ -1,19 +1,19 @@
 package:
   name: manorm
-  version: 1.1.3
+  version: 1.1.4
 
 source:
-  url: https://pypi.python.org/packages/b3/fd/fcec99d7f9ef6aa6b61253faec9724ebd9b20b4641cd5e03c36515013ae2/MAnorm-1.1.3.tar.gz
-  md5: 576992d0304b529611646d7388f8ab08
+  url: https://files.pythonhosted.org/packages/25/ba/83cc20feb4cae2e9838c17841e00cb128cfc90de73102bfbea185be15329/MAnorm-1.1.4.tar.gz
+  sha256: 17504e57cff9d913ba0e3a0f57f1741d72aef5167055d46c83ad78e09ed7f638
 
 build:
-  number: 1
+  number: 0
   skip: True  # [not py27]
   entry_points:
     - manorm=manorm.cmdline:main
 
 requirements:
-  host:
+  build:
```

Comment:
`host` was the correct section name.

Could replace `skip: true` with `noarch: python` in the build section by adding `python >=3` to the requirements host and run sections.